README.md

```diff
@@ -178,16 +178,37 @@ $ foxglove data export --device-id dev_flm75pLkfzUBX2DH --start 2001-01-01T00:00
 
 #### Studio extensions
 
-With a paid foxglove account, you can upload [Studio extensions](https://foxglove.dev/docs/studio/extensions/getting-started) to share with your organization.
+With a paid foxglove account, you can upload
+[Studio extensions](https://foxglove.dev/docs/studio/extensions/getting-started)
+to share with your organization.
 
-Extensions are created and packaged with the [foxglove-extension](https://github.com/foxglove/create-foxglove-extension/) tool. The latest version of each uploaded extension will be installed in Studio for all organization members.
+Extensions are created and packaged with the
+[foxglove-extension](https://github.com/foxglove/create-foxglove-extension/)
+tool. The latest version of each uploaded extension will be installed in Studio
+for all organization members.
 
 To publish a new extension, or update one with a newer version:
 
 ```
 foxglove extensions upload ./my-extension.1.0.0.foxe
 ```
 
+To list your extensions:
+
+```
+foxglove extensions list
+```
+
+You can use the global `--format` flag to change the output type.
+
+To unpublish an extension, use the ID listed from the above command. This will
+**delete** your extension files and cause the extension to be uninstalled from
+Studio for all organization members.
+
+```
+foxglove extensions unpublish ext_BsGXKGsZ9c4WQF1
```

Comment:
similarly, wonder if this returns output we could show

Reply:
We document the API endpoint as returning 200; the body isn't very meaningful. But I can print something friendly like we do for uploads. On the other hand, I notice that our list endpoints output nothing when receiving empty results. Should I consider the default output format to be intended for human consumption?

Reply:
from slack - I wasn't thinking of scripting so much as just consistency, since I remembered the upload command printed something (and does that bar thing). If we wanna return a zero exit and no printing I don't have a super strong objection, though confirmation also feels nice. Lots of tools will just print nothing on success, but I could think of some human friendly things we could do like a message or a correlation ID.